<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QWidget" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="FrameStyleSheet">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background:rgba(64,64,64,60%);
    border: 1px solid black;
}

QPushButton
{
	background-color: rgb(136, 136, 136);
	color: white;
	border: 1px solid black;
	padding: 2px;
}


QPushButton:pressed 
{
    background-color: rgb(64, 64, 64);
}

QLabel
{
	color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="buttonVertLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="toolNameLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Browser</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="wikiPushButton">
          <property name="text">
           <string>Wiki</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="snowcrowsPushButton">
          <property name="text">
           <string>Snow Crows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gw2efficiencyPushButton">
          <property name="text">
           <string>GW2 Efficiency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="youtubePushButton">
          <property name="text">
           <string>Youtube</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="backPushButton">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forwardPushButton">
          <property name="text">
           <string>Forward</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshPushButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="webViewLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="pageLoadingProgressBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>5</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWebEngineView" name="webEngineView">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="url">
        <url>
         <string>https://wiki.guildwars2.com/wiki/Main_Page</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
